### DISCUSSION TITLE
Shiny Background Job for a MAUI app targeting Windows

### DISCUSSION THREAD

**Discussion by Richard-Dufour** [shinyorg/shiny#1456] [Q&A]
My app runs on iOS, Android but also Windows. 

Is there a plan to support Shiny Jobs on Windows in the future?

Right now I have an if iOS or Android then use Shiny else if windows just run straight up (no job) but sure would be nice to have the support for Windows.

**Comment by aritchie** [comment]
I have no interest in supporting windows.  If I offer something in the future, it will likely be a paid addon

**Reply by Richard-Dufour** [reply]
That would be more than fair as you already provide Shiny for free to the community. 

Thanks Allan
